packages/filebrowser/src/listing.ts

```diff
@@ -3562,6 +3564,27 @@ namespace Private {
       return 0;
     }
 
+    /**
+     * Compare two items by their name using `translator.languageCode`, with fallback to `navigator.language`.
+     */
+    function compareByName(a: Contents.IModel, b: Contents.IModel) {
+      const languageCode = translator.languageCode.replace('_', '-');
```

Comment:
It looks like sometimes `translator.languageCode` will be undefined. It should not be but it looks like the tests are failing because of that:

```
Parsed an error from text content: undefined TypeError: undefined is not an object (evaluating 'translator.languageCode.replace')
```

Maybe this would help?

```suggestion
      const languageCode = (translator.languageCode ?? navigator.language).replace('_', '-');
```